### DISCUSSION TITLE
Actions: Why does label() not get rendered when disabled() is true?

### DISCUSSION THREAD

**Discussion by flolanger** [filamentphp/filament#11636] [General]
I noticed that when an action is disabled, there is **no way** to tell the user why it is disabled. Somehow, the `label()` doesn't render when `disabled()` is true. Why is that? Normally, the user expects some hint when hovering over the disabled element.

Can we maybe introduce or discuss a change that makes this possible?

```php
Action::make('importArticles')
  ->icon('heroicon-o-arrow-path')
  ->disabled()
  ->label('Some text to show when disabled.') // Label doesn't show
  ->action(...)
  ```

Thank you!

**Comment by flolanger** [comment]
Followup:
Seems like the label does get rendered, but the inline-style `"disabled:pointer-events: none"` prevents the label from showing up. Removing this would be the solution. Is there any reason why it should be used? Without it, the button is still disabled.

**Reply by leandrocfe** [reply]
Do you mean, tooltip?

**Reply by flolanger** [reply]
Discussed with Dan in discord:
It seems that neither tooltip nor label / title tag is working if ->disabled() is set.

`pointer-events: none` comes from this PR, but there is no reasoning behind unfortunatly:
https://github.com/filamentphp/filament/pull/2705

**Reply by wychoong** [reply]
TBH I don’t remember what is the exact issue. It could be cursor not showing the correct  🚫 symbol for the disabled button. And at that time we might not have that many features (eg: tooltip) that your issue was not detected. 

Feel free to PR what can fix your issue and we will see after that.
